CHANGELOG.txt

```diff
@@ -67,6 +67,7 @@ master:
     * added read support for FOCMEC 'out' and 'lst' files (see #2156)
  - obspy.signal.trigger:
     * fix a bug in AR picker (see #2157)
+    * function added (see #2341)
```

Comment:
this got garbled up somehow, I think you deleted a line?

Reply:
Yep I did, accidentally ... 
pushing the updated one now